tests/tests/cpld_regional_natl_c192

```diff
@@ -0,0 +1,72 @@
+###############################################################################
+#
+#  FV3 CCPP regional c192 test for North Atlantic domain
+#
+###############################################################################
+
+export TEST_DESCR="FV3 CCPP regional c192 for North Atlantic domain coupled with HYCOM"
```

Comment:
Same here (and all other test files below)

Reply:
@climbfuji, Removed accordingly. Also please note the new added HAFS regression test names have been updated. Thanks!